module/input/SensorFilter/src/SensorFilter.cpp

```diff
@@ -87,6 +84,15 @@ namespace module::input {
         return s.str();
     }
 
+    [[nodiscard]] std::string servoErrorString(uint32_t id, RawSensors::Servo servo, const uint& errorCode) {
+        std::string servoName(static_cast<ServoID>(id));
+        std::stringstream s;
+        s << makeErrorString(servoName, errorCode);
```

Comment:
Should we delineate where the string from `makeErrorString` ends and this info starts more clearly? I am just thinking I would probably appreciate a clear border between the part that's tell me what the problem is and the diagnostic info added by this function

Reply:
I think it is pretty clear as it is right now

Reply:
I misunderstood what you were saying and I have made a change for what you said :)